src/snowflake/snowpark/_internal/data_source/__init__.py

```diff
@@ -0,0 +1,3 @@
+#
+# Copyright (c) 2012-2025 Snowflake Computing Inc. All rights reserved.
+#
```

Comment:
nit:

you can export DataSourcePartitioner as a module class, check out the snowflake.snowpark.__init__.py